career/ladder.md

```diff
@@ -53,10 +57,20 @@ Here are some examples of other career ladders:
 - [ ] Can write code in at least one language.
 - [ ] Broad knowledge of main CS + XP concepts.
 
-### Level 2
+### Level 2 - Software Engineer
 
 #### Impactful
 - [ ] Capable of providing support for their area, including systems they are not familiar with.
+- [ ] Understands and can provide a high-level overview of the current milestone
+  of a project.
+- [ ] Focuses on building strong relationships with their team, direct manager,
+  and project manager/product owner.
+- [ ] Identifies unclear requirements and reaches out to relevant stakeholders
+  to clarify issues.
+- [ ] Reliably do what they say they will, and communicates promptly if an issue
+  arises.
+- [ ] Requires minimal guidance from seniors and can help unblock juniors when
+  necessary.
```

Comment:
Is this a level 2 or level 3 attribute?

Reply:
I think you're right that it should perhaps be moved which I've done now.